### DISCUSSION TITLE
"latex-workshop.hover.preview.newcommand.newcommandFile" what's the format of this newcommandfile?

### DISCUSSION THREAD

**Discussion by tyehuoPHD** [James-Yu/LaTeX-Workshop#3775] [Q&A]
# "latex-workshop.hover.preview.newcommand.newcommandFile" what's the format of this newcommandfile?

i want to type this file. but i cant find examples. please help me.

**Comment by jlelong** [comment]
https://github.com/James-Yu/LaTeX-Workshop/wiki/Hover#latex-workshophoverpreviewnewcommandnewcommandFile

The file should be a valid LaTeX file containing `\newcommand` definitions.
